source/eventHandler.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, eventName, obj, kwargs):
 		self.kwargs = kwargs
 		self._gen = self.gen(eventName, obj)
 		try:
-			self.next()
+			next(self)

```

Comment:
I'm afraid this is subject to serious breakage as it is now. You're calling next(self) instead of self.next(). However, next is a method on self. Here, you're not dealing with a real iterator, but with a class that only inherits from object.

I'd like to suggest to revert these changes to eventHandler altogether.